catalog/JRC/JRC_D5_EUCROPMAP_V1.jsonnet

```diff
@@ -23,20 +23,21 @@ local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
     ee_const.ext_ver,
```

Comment:
please also change the line
  '2018-01-01T00:00:00Z', '2019-01-01T00:00:00Z'),
to
  '2018-01-01T00:00:00Z', null),
  
  (leaving end date as null is more flexible, as it will make our scripts query for the actual end date).

Reply:
Hi Simon, thanks for the approval of the changes. Do you know when can we expect to see them reflected in the catalogue?